tests/test__collection_api.py

```diff
@@ -4269,6 +4269,149 @@ def test__aggregate_filter_wrong_options(self):
                 self.db.collection.aggregate(
                     [{'$project': {'filtered_items': {'$filter': option}}}])
 
+    def test__aggregate_slice(self):
+        self.db.collection.drop()
+        collection = self.db.collection
+        self.db.collection.insert_many([
+            {
+                '_id': 0,
+                'items': list(range(10)),
+            },
+            {
+                '_id': 1,
+                'items': list(range(10, 20)),
+            },
+            {
+                '_id': 2,
+                'items': list(range(20, 30)),
+            },
+        ])
+
+        empty = [
```

Comment:
I suggest we write those tests in a more compact manner:
```py
empty = [
  {'_id': 0, 'slice': []},
  {'_id': 1, 'slice': []},
  {'_id': 2, 'slice': []},
],
```
5 lines instead of 17

Reply:
done